crates/egui/src/widgets/plot/items/mod.rs

```diff
@@ -670,6 +722,12 @@ impl Text {
         self
     }
 
+    // Allowed hovering this line in the plot.
```

Comment:
```suggestion
    // Allowed hovering this item in the plot. Default: `true`.
```

here and everywhere

Reply:
Thank you! I thought I fixed it.

Reply:
Thank ! I also added the missing slash for the doc.